packer/ansible/rackhd_source.yml

```diff
@@ -0,0 +1,25 @@
+---
+- name: Development Server
+  hosts: local
+  sudo: no
+  roles:
+    - apt
+    - build
+    - git
+    - rabbitmq
+    - mongodb
+    - node
+    - isc-dhcp-server
+    - monorail
```

Comment:
I still don't have the history, if @heckj happens to see this comment, it will be very appreciate to give some info why include the `monorail` here and what is it?

Or whether @panpan0000 and @PengTian0 know some detail?